bin/smarthome.py

```diff
@@ -279,6 +280,12 @@ def start(self):
                 except Exception as e:
                     self.logger.exception("Problem reading {0}: {1}".format(item_file, e))
                     continue
+            if item_file.endswith('.yaml'):
+                try:
+                    item_conf = lib.config_yaml.parse(self._items_dir + item_file, item_conf)
+                except Exception as e:
+                    self.logger.exception("Problem reading {0}: {1}".format(item_file, e))
+                    continue
```

Comment:
This will only support yaml format for the items configuration files. I think we should support it on all configuration files. Few lines above the env files are read, but not supported to be in yaml file.

These two sections could be combined into one method and used in both places.